catalog-info.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: jekyll-assets
+  annotations:
+    jira/project-key: MKTG
+spec:
+  type: service
+  lifecycle: production
+  owner: marketing
```

Comment:
I'm not familiar with what this is used for, but "marketing" feels overly broad to me? The more specific team would be "web and content"

Reply:
Thanks @vallieres 

Can there be multiple teams listed here?
I think the best fit is [`web-development-optimization`](https://backstage.tools.circleci.com/catalog/default/group/web-development-optimization)
That team recently got combined with [`content`](https://backstage.tools.circleci.com/catalog/default/group/content), though this doesn't appear to be reflected in that list yet.

Reply:
I think it can only be one. I will that updated with `web-development-optimization`.